exporters/otlp/include/opentelemetry/exporters/otlp/otlp_grpc_exporter.h

```diff
@@ -23,6 +23,8 @@ namespace exporter
 namespace otlp
 {
 
+class OtlpGrpcClient;
```

Comment:
Can this be moved to `otlp_grpc_exporter_options.h` as a central place?

Reply:
Users use factory and `otlp_grpc*_options.h` to create OTLP gRPC exporters, but I think users do not need know the internal class `OtlpGrpcClient`?